### PR TITLE
[Resnet] Sync resnet app with validated model

### PR DESCRIPTION
**Pull request by zhoonit**
- [Resnet] Sync resnet app with validated model

```
This patch syncs resnet application with validated model.
Changes include:

1. s/setIteration/updateIteration
2. normalize input
3. fixed batchsize of 128 -> args received from the sysargs
4. output layer to copy
5. silent failed to read epoch idx and iteration

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

File: nntrainer/models/neuralnet.cpp [hunk 1]
```diff
@@ -430,10 +430,14 @@ void NeuralNetwork::readModel() {
...
-              "[NeuralNetwork::readModel] failed to read epoch_idx");
-  checkedRead(model_file, (char *)&tmp.iter, sizeof(iter),
-              "[NeuralNetwork::readModel] failed to read iteration");
+  try {
+    checkedRead(model_file, (char *)&tmp.epoch_idx, sizeof(epoch_idx),
+                "[NeuralNetwork::readModel] failed to read epoch_idx");
+    checkedRead(model_file, (char *)&tmp.iter, sizeof(iter),
+                "[NeuralNetwork::readModel] failed to read iteration");
+  } catch (...) {
+    std::cerr << "failed to read epoch idx, proceeding with default index\n";
```
Comment (kparichay) [inline nntrainer/models/neuralnet.cpp:439]: please add a comment here that this failure is allowed at the end of file read. If this has failed, no other read operation should be performed on this file.
Or maybe move model_file.close() inside try and catch both to emphasize on it.

**Review by kparichay — APPROVED** [review]
LGTM other than the minor comment

File: nntrainer/models/neuralnet.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/models/neuralnet.cpp:439]: Good idea, added :)

**Review by jijoongmoon — APPROVED** [review]
LGTM with one miner. 
. Update the document with some reuslts. ( later PR, I guess )

**Comment by jijoongmoon** [comment]
I wonder that validated model means keras model ? If user download the keras model and then, is this model configuration same with it?

**Comment by zhoonit** [comment]
yes, they are in the internal repo. #904

I'll update those two in a separate pr.

**Pull request merged**